lockbox-ios/Common/Extensions/UIDevice+.swift

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+extension UIDevice {
+    public static var oniPhoneX: Bool {
```

Comment:
Two things: 

 * nit: naming. Prefer `is` preposition to `on`.
 * would prefer to do feature detection rather than model detection, i.e. `hasFaceID` rather than `isiPhoneX`.